chrome/content/zotero/zoteroPane.js

```diff
@@ -4135,7 +4160,8 @@ var ZoteroPane = new function()
 				(target) => {
 					// can't copy collection into itself or into non-editable groups
 					return selected == target
-						|| (target instanceof Zotero.Group && !target.editable);
+						|| (target instanceof Zotero.Group && !target.editable)
+						|| linkedCollectionsExist[target.libraryID];
```

Comment:
Is this supposed to be here? Is this not taken care of by the block above, which runs asynchronously?

Reply:
No, sorry about that. This was accidentally leftover from when I was trying to not disable menus asynchronously.